deployments/continuous-deployment-config/ocis_traefik/released.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: continuous-deployment-ocis-traefik-released
+  server:
+    server_type: cx21
+    image: ubuntu-20.04
+    location: nbg1
+    inital_ssh_key_names:
```

Comment:
```suggestion
    initial_ssh_key_names:
```

Reply:
also fixed in https://github.com/owncloud-devops/continuous-deployment/commit/b35286d1f8c2fc40d14ceef74580a24fa70ebb70